app/models/statistics/total_savings.rb

```diff
@@ -0,0 +1,27 @@
+class Statistics::TotalSavings
+  def to_s
+    if completed_auctions.count > 0
+      Currency.new(total_starting_price - total_winning_bid_amount).to_s
+    else
+      'n/a'
+    end
+  end
+
+  private
+
+  def total_starting_price
```

Comment:
Yep, the math looks good to me. Mathematical average which is just one "measure of central tendency" and in the future we can change this if another gives better business intelligence.